### ISSUE TITLE
v14.3-rc - umb-code-block displays json flat 

### ISSUE THREAD

**Issue by KevinJump** [umbraco/Umbraco-CMS#17152]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v14.3.0-rc

### Bug summary

When using the <umb-code-block> component to display a json snippet, the snippit is now displayed flat. because of a change from pre to pre-line in the styles. 

### Specifics

The commit ( https://github.com/umbraco/Umbraco.CMS.Backoffice/commit/68c8956ecc6b6d47248efdf64b16c072aba1fd31 ) changes the white-space styling of the code-block's pre value from pre to pre-line and this means the white space used when indenting JSON is no longer shown,. 

v14.2 
![image](https://github.com/user-attachments/assets/d914762a-3520-4fa6-998c-fecf7a459e06)

v14.3. 
![image](https://github.com/user-attachments/assets/88047ed3-899d-468f-b499-4166bbfe3797)

in these examples code is. 

```ts
<umb-code-block language="JSON" .copy=${true}
	>${JSON.stringify(keyJson, null, 1)}</umb-code-block
>
```

### Steps to reproduce

add a `<umb-code-block>` to a component with some stringyfied json in it. (as above).

### Expected result / actual result

would be ideal if it was indented - or if there was a way to override the white-space style via an attribute on the component?

**Comment by iOvergaard** [comment]
Thanks for the report, @KevinJump. I can see my fix here wasn't ideal. Let me have another look at it!

**Comment by iOvergaard** [comment]
Turns out we didn't need this change at all, so I have reverted it. The fix will be included in the final release of 14.3.
